ruby/ql/lib/codeql/ruby/controlflow/CfgNodes.qll

```diff
@@ -147,10 +147,10 @@ abstract private class ExprChildMapping extends Expr {
    * Holds if `child` is a (possibly nested) child of this expression
    * for which we would like to find a matching CFG child.
    */
-  abstract predicate relevantChild(Expr child);
+  abstract predicate relevantChild(AstNode child);
```

Comment:
Should this class now  be called `AstChildMapping` instead of `ExprChildMapping`?

Reply:
I think it is OK to leave as is, since the class extends `Expr`.